client/src/Movies/Movie.js

```diff
@@ -1,17 +1,20 @@
 import React, { Component } from 'react';
 import axios from 'axios';
+import MovieCard from './MovieCard';
+
 
 export default class Movie extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      movie: null
+      movie: null,
```

Comment:
Although this would be normally ok, make the habit of setting the initial value the data type you will be using. For example, if you are going to set the state with an array of objects, initialize with an empty array. And an empty string if you're expecting a string. Only do null if you're going to expect a number.